tests/test_utils.py

```diff
@@ -119,3 +120,17 @@ class Foo:
     pass
 """
     check_decorator_names(code, ["@foo", "@bar.yz"])
+
+
+@pytest.mark.skipif(
+    sys.version_info < (3, 9), reason="requires python3.9 or higher"
+)
```

Comment:
the only thing problematic for older versions is the test, so we ignore this specific test